cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -0,0 +1,258 @@
+package com.cloudinary.test;
+
+
+import com.cloudinary.Cloudinary;
+import com.cloudinary.api.ApiResponse;
+import com.cloudinary.provisioning.Account;
+import org.junit.*;
+import org.junit.rules.TestName;
+
+import java.util.*;
+
+import static java.util.Collections.emptyMap;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public abstract class AbstractAccountApiTest extends MockableTest {
+    private static Random rand = new Random();
+    protected Account account;
+    private static Set<String> createdSubAccountIds = new HashSet<String>();
+    private static Set<String> createdUserIds = new HashSet<String>();
+    private static Set<String> createdGroupIds = new HashSet<String>();
+
+    @BeforeClass
+    public static void setUpClass() {
+
+    }
+
+    @Rule
+    public TestName currentTest = new TestName();
+
+    @Before
+    public void setUp() throws Exception {
+        System.out.println("Running " + this.getClass().getName() + "." + currentTest.getMethodName());
```

Comment:
@nitzanj please remove

Reply:
This is not a debug log, that's what ends up in the gradle tests standard output, it's required.